README.md

```diff
@@ -394,7 +565,17 @@ while results.next_token:
 
 The `next_token` returned from a query will be 0 if there are no more records to return. `total_results` represents the total number of records in e3db that match the executed query. 
 
-See [the integration tests](e3db/tests/test_search_integration.py) for more examples.
+See [pagination example](examples/simple_paginate_results.py) for full code example.
+
+#### Search Count Restraints
+
+You aren't limited to a specific number of searches, however for single a single search the maximum amount of records returned in a single result page is 1000, and exceeding this limit will by reject your request. Pagination as shown above can be used to grab more than 1000 records.
```

Comment:
`for a single search`

Reply:
Need to clarify what limit is being exceeded, assuming you are talking about the limit parameter, but the wording is ambiguous/could be taken to mean that the users search could return more than 1000 per page of paginated results(which it can't)